templates/certificates/dynamic-styles.php

```diff
@@ -23,15 +23,15 @@
 $gfonts_preconnet = false;
 ?>
 
-<!-- Certificates Dynamic Styles -->
+<!-- Certificates  -->
 <?php foreach ( $fonts as $font ) : ?>
 	<?php if ( ! empty( $font['href'] ) ) : ?>
 		<?php
-		if ( ! $gfonts_preconnet && false !== strpos( $font['href'], 'fonts.googleapis.com' ) ) :
+		if ( ! $gfonts_preconnet && false !== strpos( $font['href'], LLMS_PLUGIN_URL . 'assets/css/localstyles.css'  ) ) :
 			$gfonts_preconnet = true;
 			?>
-			<link rel="preconnect" href="https://fonts.googleapis.com">
-			<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+			<link rel="preconnect" href="<?LLMS_PLUGIN_URL ?>">
+			<link rel="preconnect" href="<?LLMS_PLUGIN_URL ?>" crossorigin>
```

Comment:
With the other changes I've suggested for loading methodology we should keep this to preconnect to google. That way if a use opts in to loading from google directly (via a filter) the preconnect will still happen.